bam_mate.c

```diff
@@ -156,24 +156,28 @@ static bool plausibly_properly_paired(bam1_t* a, bam1_t* b)
         return false;
 }
 
-static void bam_format_cigar(const bam1_t* b, kstring_t* str)
+// Returns true on error, false on success
```

Comment:
I don't object but for some reason Github thinks you're commenting on an outdated changeset? No clue how that is even possible, I haven't touched it for a week.